### DISCUSSION TITLE
KafkaConnect image build if not exist

### DISCUSSION THREAD

**Discussion by ltson4121994** [strimzi/strimzi-kafka-operator#9432] [Q&A] [answered]
Hi, I am a newbie to strimzi and I wonder if there are any ways I could specify the image spec and if the image does not exist, I could specify how I could build the image. I have looked at the examples and most of them do not specify both "image" and "build" spec at the same time so I am not sure if this is possible.

**Comment by scholzj** [comment, marked as answer]
Strimzi will never know if the image exists. That would include a complicated querying of the container repository etc. And even if we queried it, we would not know if it corresponds to the build specification or not. So this is not possible.

**Reply by ltson4121994** [reply]
I see, thanks for the answer.
